### DISCUSSION TITLE
Package Readme?

### DISCUSSION THREAD

**Discussion by IgorDePaula** [community/community#72999] [Packages] [answered]
### Select Topic Area

Question

### Body

For I have a readme on one of my package I need link it to a one repository, but, if my repository is private and my package public, I don`t see the readme file/ page on package link, only if both are public.

**Comment by mhaemnn** [comment, marked as answer]
### maybe one way: 
**Document Separately**
Create separate documentation that is public, for example, using the Wiki service on GitHub or creating a dedicated repository for documentation. Link this documentation in the package description or in a special section within the package.

**Reply by IgorDePaula** [reply]
Dedicated repository for documentation, good idea, thanks.

**Reply by mhaemnn** [reply]
You're welcome

enthusiastic and enjoy coding
